source/intel.js

```diff
@@ -1384,20 +1404,36 @@ function NeptunesPrideAgent() {
     npui.SideMenuItem("icon-eye", "n_p_a", "trigger_npa").roost(npui.sideMenu);
 
     let superFormatTime = Crux.formatTime;
-    let relativeTimes = true;
+    let relativeTimes = 0;
     Crux.formatTime = function (ms, mins, secs) {
-      if (relativeTimes) {
-        return superFormatTime(ms, mins, secs);
-      } else {
-        return msToEtaString(ms, "");
+      switch (relativeTimes) {
+        case 0: //standard
+          return superFormatTime(ms, mins, secs);
+        case 1: //ETA, - turn(s) for turnbased
+          if (!NeptunesPride.gameConfig.turnBased) {
+            return msToEtaString(ms, "");
+          } else {
+            return (
+              superFormatTime(ms, mins, secs) +
+              " - " +
+              (
+                ((ms / 3600000) * 10) /
+                NeptunesPride.universe.galaxy.tick_rate
+              ).toFixed(2) +
+              " turn(s)"
+            );
+          }
```

Comment:
IF its turn based only then it will use your tick format. let me try this out now.

Reply:
Wow it looks great! Thank you this is actually relevant compared to the alternative. A great update!